src/telemetry/metrics.yaml

```diff
@@ -0,0 +1,52 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+---
+$schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
+
+app_events:
+  page_view:
+    type: event
+    lifetime: ping
```

Comment:
@rosahbruno can this be left out? If so, let's leave this out, it's the only allowed value :)